python/pyarrow/tests/parquet/test_basic.py

```diff
@@ -76,20 +76,16 @@ def test_set_data_page_size():
         _check_roundtrip(t, data_page_size=target_page_size)
 
 
-@pytest.mark.pandas
 def test_set_write_batch_size():
```

Comment:
This now runs when Pandas is not present, which is great, but fails when numpy is not present.
Can you try adding `@pytest.mark.numpy`?